libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m

```diff
@@ -156,7 +158,10 @@ - (void)viewDidLoad {
     [super viewDidLoad];
     [self setEdgesForExtendedLayout:UIRectEdgeNone];
     
-    [self.tableView registerClass:[UITableViewCell class] forCellReuseIdentifier:SFDCLoginHostListCellIdentifier];
+    // TODO: Remove the check once we move to iOS 9 as minimum.
```

Comment:
iOS 9 is already the min version (see https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1749). What did you mean?